_includes/field-description-vatamount.md

```diff
@@ -3,14 +3,14 @@
 {% assign currency = "SEK" %}
 {% endif %}
 {%- capture vatamount_text -%}
-The payment's VAT (Value Added Tax) `amount`, entered in the lowest monetary 
+The payment's VAT (Value Added Tax) `amount`, entered in the lowest monetary&nbsp;
 unit of the selected currency. E.g.:&nbsp;
 **`10000`** = `100.00` {{ currency }},&nbsp;
 **`5000`** = `50.00` {{ currency }}.&nbsp;
 
-The `vatAmount` entered will not affect the `amount` shown on the payment page, 
-which only shows the total `amount`. This field is used to specify how much of
-the total `amount` the VAT will be. Set to `0` (zero) if there is no VAT 
+The `vatAmount` entered will not affect the `amount` shown on the payment page,&nbsp;
+which only shows the total `amount`. This field is used to specify how much of&nbsp;
+the total `amount` the VAT will be. Set to `0` (zero) if there is no VAT&nbsp;
```

Comment:
Is the `&nbsp;` really necessary here? Is there no alternative that does not look so disruptive when reading the markdown?

Reply:
>Is the `&nbsp;` really necessary here? Is there no alternative that does not look so disruptive when reading the markdown?

Nope, good point. I got rid of it with some crazy filtering magic in e7221bb4de5fa7b8f0b23a72ae1a5dac131f10a8.